<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Email Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="2"
           time="-1650287169,775">
  <properties />
            <testcase name="preCondition" time="0,000" classname="MailTest">
                <failure type="java.lang.NullPointerException"
                  message="(null)"
              >
<![CDATA[java.lang.NullPointerException
  driver.BrowserInstance.createInstance(BrowserInstance.java:9)
  BaseTest.preCondition(BaseTest.java:21)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:566)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
  org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
  org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
  org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
  org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:683)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:510)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
  org.testng.TestRunner.privateRun(TestRunner.java:770)
  org.testng.TestRunner.run(TestRunner.java:591)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
  org.testng.SuiteRunner.run(SuiteRunner.java:304)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
  org.testng.TestNG.runSuites(TestNG.java:1032)
  org.testng.TestNG.run(TestNG.java:1000)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
  org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
      </failure>

          </testcase>
  
            <testcase name="postCondition" time="0.0" classname="MailTest">
          <skipped />
    </testcase>
            <testcase name="newMailWithoutRecipientTest" time="0.0" classname="MailTest">
          <skipped />
    </testcase>
  
  
</testsuite>
